spec/features/skills_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+describe "Skills", js:true, type: :request do
+  it "adds a skill to a new user" do
+    visit root_path
+    click_link "New User"
+    fill_in "First name", with: 'Chiquita'
+    fill_in "Last name", with: 'Banana'
+    page.execute_script("$('#user_last_name').trigger('change')")
+    click_link "Add a skill"
+    fill_in "Skill", with: 'Dancing'
+    find_field('Level').find('option[value="3"]').click
+    click_button "Create User"
+    expect(page).to have_content("User was successfully created.")
+  end
+
+  it 'edits a skill on an existing user' do
+    visit root_path
+    expect(page).to have_text("Keanu")
+    first(:link, 'Show').click
```

Comment:
Problem here: I want to use jquery elements here to make sure i'm clicking on the right person rather than hoping that Keanu Reeves is the first person on this list of users.